lighthouse-core/gather/gather-runner.js

```diff
@@ -409,6 +418,7 @@ class GatherRunner {
           passConfig,
           // *pass() functions and gatherers can push to this warnings array.
           LighthouseRunWarnings: baseArtifacts.LighthouseRunWarnings,
+          firstPass,
```

Comment:
now that we're passing this variable around to other places, WDYT about renaming to `isFirstPass` or something so we know it's a bool and not a ref to the first pass config or something?